### PR TITLE
refactor: replace menus and account switcher with bloom-ui menu

### LINKED ISSUE #733: 🛠 - Replace Menus with bloom-ui equivalent

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/menus/CollectibleDetailsMenu.svelte [hunk 1]
```diff
@@ -66,42 +39,65 @@
...
+            props: {
+                title: localize('actions.confirmNftBurn.title', {
+                    values: {
+                        nftName: nft.name,
+                    },
+                }),
+                description: localize('actions.confirmNftBurn.description'),
+                hint: localize('actions.confirmNftBurn.hint'),
+                warning: true,
+                confirmtitle: localize('actions.burn'),
```
Comment (maxwellmattryan) [inline packages/desktop/components/menus/CollectibleDetailsMenu.svelte]: `confirmTitle`?

File: packages/desktop/components/menus/GovernanceDetailsMenu.svelte [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<script lang="typescript">
+    import { IconName, Menu } from '@bloomwalletio/ui'
+    import { localize } from '@core/i18n'
+    import { PopupId, openPopup } from '../../lib/auxiliary/popup'
```
Comment (maxwellmattryan) [inline packages/desktop/components/menus/GovernanceDetailsMenu.svelte]: We should use a path alias for this, `@desktop/auxiliary/popup` I think will work?

File: packages/desktop/components/modals/index.js [hunk 3]
```diff
@@ -1,3 +1,2 @@
-export { default as AccountSwitcherModal } from './AccountSwitcherModal.svelte'
 export { default as FilterModal } from './FilterModal.svelte'
 export { default as ProfileActionsModal } from './ProfileActionsModal.svelte'
```
Comment (maxwellmattryan) [inline packages/desktop/components/modals/index.js:2]: Did we decide not to implement this one yet since it got moved to profile selection?

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
Have some Qs / comments

File: packages/desktop/components/menus/CollectibleDetailsMenu.svelte (hunk 1 above)
Comment (nicole-obrien) [reply packages/desktop/components/menus/CollectibleDetailsMenu.svelte]: ctrl+f issue 🤦‍♀️

File: packages/desktop/components/modals/index.js (hunk 3 above)
Comment (nicole-obrien) [reply packages/desktop/components/modals/index.js:2]: In scope of sidebar, there will still be a separate profile menu on selection page

**Review by maxwellmattryan — APPROVED** [review]
LGTM :cherry_blossom:

**Pull request merged**